main.py

```diff
@@ -42,25 +58,29 @@ def prompt_redo():
 
 
 print("At any time, type EXIT to stop. ")
-while True:
-    mode = str(input("What mode do you wish to use? FLIP == flip text, CASE == change casing. "))
+while i < 1:
```

Comment:
```suggestion
while True:
```

We don't need to use i < 1.